presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveTransactionalTable.java

```diff
@@ -107,7 +108,7 @@ public void testReadInsertOnly(boolean isPartitioned, BucketingType bucketingTyp
             throw new SkipException("Presto Hive transactional tables are supported with Hive version 3 or above");
         }
 
-        String tableName = "test_insert_only_table_read";
+        String tableName = format("test_insert_only_table_read_%s", randomTableSuffix());
```

Comment:
per code style, we do not use format when there is only trailing %s, concat instead